addons/website/data/website_data.xml

```diff
@@ -57,7 +57,7 @@
                                     </div>
                                 </div>
                             </div>
-                            <div class="oe_structure mt-2"/>
+                            <div class="oe_structure mt-2 oe_structure_none" data-editor-message=""/>
```

Comment:
Clearly, we don't want to have to add `data-editor-message=""` everywhere, I think this is the opposite that we want.